packages/amazonq/src/api.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GenerateAssistantResponseCommandOutput, GenerateAssistantResponseRequest } from '@amzn/codewhisperer-streaming'
+import { AuthUtil } from 'aws-core-vscode/codewhisperer'
+import { ChatSession } from 'aws-core-vscode/codewhispererChat'
+import { api } from 'aws-core-vscode/amazonq'
+
+export default {
```

Comment:
If it wasn't a circular dependency, this would `extend` the interface defined in `packages/core/src/amazonq/extApi.ts` , right?

This is kind of strange. Somewhat unavoidable since most of amazonq lives in "core" currently.